### PR TITLE
Fixing some issues

### LINKED ISSUE #3: Consider making this class static or a readonly record/struct
https://github.com/barnesyard/TargetestGame/blob/2fbe6880c753d0a085037e58dec7eb79fcfbbb40/TargetsGameApp/Program.cs#L28

### PR DESCRIPTION
**Pull request by barnesyard**
fixes #3 #4 #2
@mrjavaguy I made changes (added this text to see how to tag someone, it didn't complete for me)

### CODE REVIEW

File: GenerateGridJson/GenGridProgram.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ static void Main(string[] args)
     {
         Console.WriteLine("Reading CSV and converting to JSON...");
 
-        string csvPath = "c.csv";
+        string csvPath = "Game From 2024-10-21.csv";
```
Comment (mrjavaguy) [inline GenerateGridJson/GenGridProgram.cs:25]: how about using the args, so the person running can have different files.

File: GenerateGridJson/grid.json [hunk 2]
```diff
@@ -1,7 +1,5 @@
 [
```
Comment (mrjavaguy) [inline GenerateGridJson/grid.json:1]: should this file even be part of the PR?

File: GenerateGridJson/GenGridProgram.cs (hunk 1 above)
Comment (barnesyard) [reply GenerateGridJson/GenGridProgram.cs:25]: yes, I was focusing on initial functionallity with this commit. I will open it up to user input with a future PR. I won't do this as part of this PR.

File: GenerateGridJson/grid.json (hunk 2 above)
Comment (barnesyard) [reply GenerateGridJson/grid.json:1]: It makes it easier for a person who is trying to build and run locally. I could move it to a "samples" folder or something like that.

**Pull request merged**